src/fusor/translator.py

```diff
@@ -234,10 +279,14 @@ async def from_jaffa(
         )
 
         return self._format_fusion(
-            gene_5prime_element,
-            gene_3prime_element,
-            tr_5prime,
-            tr_3prime,
+            fusion_partners.gene_5prime_element,
+            fusion_partners.gene_3prime_element,
+            tr_5prime
+            if isinstance(fusion_partners.gene_5prime_element, GeneElement)
+            else [None],
```

Comment:
Is `_format_fusion` signature correct? `tr_5prime` has type `TranscriptSegmentElement` or `None`, but here you have list?

Reply:
Same for `tr_3prime`